### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by timcappalli**
Adds a PR template similar to CredMan & DC API

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Closes #???
+
+The following tasks have been completed:
+
+- [ ] Modified Web platform tests (link)
+
+Implementation commitment:
+
+- [ ] WebKit (link to issue)
+- [ ] Chromium (link to issue)
```
Comment (nsatragno) [inline .github/pull_request_template.md:10]:
```suggestion
- [ ] Chromium ([link to issue](https://issues.chromium.org/issues/new?component=1456855&template=0))
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Closes #???
+
+The following tasks have been completed:
+
+- [ ] Modified Web platform tests (link)
+
+Implementation commitment:
+
+- [ ] WebKit (link to issue)
```
Comment (nsatragno) [inline .github/pull_request_template.md:9]:
```suggestion
- [ ] WebKit ([link to issue](https://bugs.webkit.org/))
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+- [ ] Chromium (link to issue)
+- [ ] Gecko (link to issue)
```
Comment (nsatragno) [inline .github/pull_request_template.md:11]:
```suggestion
- [ ] Gecko ([link to issue](https://bugzilla.mozilla.org/home))
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+Documentation and checks
+
+- [ ] Affects privacy
+- [ ] Affects security
+- [ ] Pinged MDN
```
Comment (nsatragno) [inline .github/pull_request_template.md:17]: do we ever actually require this?

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+- [ ] Updated Explainer
```
Comment (nsatragno) [inline .github/pull_request_template.md:18]:
```suggestion
- [ ] Updated explainer ([link](https://github.com/w3c/webauthn/wiki)
```

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+Closes #???
+
+The following tasks have been completed:
+
+- [ ] Modified Web platform tests (link)
```
Comment (nsatragno) [inline .github/pull_request_template.md:5]:
```suggestion
- [ ] Modified Web platform tests ([link](https://github.com/web-platform-tests/wpt/))
```

**Review by nsatragno — APPROVED** [review]
Thanks! Approved with some optional suggestions.

**Review by timcappalli — COMMENTED** [review]
Adds links

File: .github/pull_request_template.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
+Closes #???
+
+The following tasks have been completed:
+
+- [ ] Modified Web platform tests ([link](https://github.com/web-platform-tests/wpt/))
```
Comment (MasterKale) [inline .github/pull_request_template.md:5]: What happens to these checkboxes in informative or editorial PR's?

Comment (timcappalli) [reply .github/pull_request_template.md:5]: You don't check them.

**Review by MasterKale — APPROVED** [review]
lgtm

File: .github/pull_request_template.md (hunk 4 above)
Comment (timcappalli) [reply .github/pull_request_template.md:19]: Not sure. I stole from DC API, which stole it from CredMan :)

**Review by akshayku — APPROVED** [review]

File: .github/pull_request_template.md (hunk 4 above)
Comment (emlun) [reply .github/pull_request_template.md:19]: If we're keeping it, is there a link we can add instructing where/how to do this?

Comment (timcappalli) [reply .github/pull_request_template.md]: removed

**Review by nsatragno — APPROVED** [review]

**Review by emlun — APPROVED** [review]

**Pull request merged**